components/brave_new_tab_ui/components/app.js

```diff
@@ -99,9 +101,7 @@ class NewTabPage extends React.Component {
     }
 
     if (this.props.newTabData.isIncognito) {
-      // TODO
-      // return <NewPrivateTab newTabData={newTabData} />
-      return null
+      return <NewPrivateTab stats={newTabData.stats} />
```

Comment:
Height needs to be adjusted here so it takes up the whole height for the background.
<img width="1198" alt="screen shot 2018-05-22 at 1 04 36 pm" src="https://user-images.githubusercontent.com/831718/40379122-51e50e3c-5dc3-11e8-823c-efb5523bf0b6.png">